api/stripe-functions.js

```diff
@@ -99,23 +100,22 @@ async function createCustomerAndSubscription(paymentMethodId, customerInfo) {
       default_payment_method: paymentMethodId,
     },
   });
-
   /* Create subscription and expand the latest invoice's Payment Intent 
    * We'll check this Payment Intent's status to determine if this payment needs SCA
    */
-  const subscription = await stripe.subscriptions.create({
+   subscription = await stripe.subscriptions.create({
     customer: customer.id,
     items: [{
       plan: customerInfo.planId,
     }],
     // trial_from_plan: true,
     expand: ["latest_invoice.payment_intent"],
+    payment_behavior:'allow_incomplete'
```

Comment:
We now need to add a `payment_behavior`, otherwise the default behavior is an error returned by stripe.
Then we cannot get any response object with `payment_intent`.

 Hence we have to add that key : `payment_behavior` that we can set to `allow_incomplete`

[Additional info here](https://stripe.com/docs/api/subscription_items/create#create_subscription_item-payment_behavior)